.github/workflows/linter.yml

```diff
@@ -2,9 +2,6 @@ name: Linter
 
 on:
   push:
-    paths:
-      - "**.cs"
-      - ".editorconfig"
```

Comment:
Does this work without any values under `push`?

Reply:
yes